### DISCUSSION TITLE
GoPro Feature Displaying on Boot

### DISCUSSION THREAD

**Discussion by mhammann57** [gopro/labs#892] [Q&A] [answered]
Is there any way to deactivate or remove the TUSB (trust USB) modification on a GoPro Hero 11 Black?  After using a QR code to make the modification, I get a text during boot that is annoying.  I thought resetting the camera and updating the firmware would deactivate the modification however, I still get the annoying text at startup. (See attached pic)  Thank you for any help/advice!  Mark
![TUSB](https://github.com/gopro/labs/assets/169004601/7215bf01-11ec-4dd6-81ba-39fa9dbe8d18)

**Comment by dnewman-gpsw** [comment, marked as answer]
If you have features activated it will be displayed.  This is to prevent users forgetting, and call support, saying something doesn't work when running some obscure hack.  So your choices:
1) !RESET <- to disable all the extensions
2) Install stock firmware
3) *DSPL=10  <- set all message displays to the shortest time ~10ms.

**Reply by mhammann57** [reply]
Worked!  Thank you dnewman!
